tidb-cloud/terraform/destroy.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Destroy resource
+summary: Learn how to destroy resource
+---
+
+# Destroy Resource
+
+To destroy the resource, you can simply use `terraform destroy` and type `yes`. Don't worry about the order of deletion, terraform will generate a DAG based on the dependencies automatically.
```

Comment:
```suggestion
To destroy resources, you can simply run the `terraform destroy` command and type `yes`. Don't worry about the order of deletion, terraform will generate a DAG based on the dependencies automatically.
```

Reply:
What is "DAG"?

Reply:
directed acyclic graph (DAG) is used to determine the dependency of resources. So, we can destroy them by order